src/status_im/ui/components/invite/chat.cljs

```diff
@@ -56,9 +56,13 @@
 (defn starter-pack []
   (let [pack        @(re-frame/subscribe [::invite/starter-pack])
         tokens      (utils/transform-tokens pack)
-        reward-text (cstr/join ", " (map (comp name :symbol first) tokens))]
+        reward-text (->> tokens
```

Comment:
We could make use of transducers here as I believe that when threading it will expand not to use transducers, if performance/memory are a concern.

Reply:
The tokens list should vary from one element to three, so we should be fine for now